lib/matplotlib/offsetbox.py

```diff
@@ -1205,6 +1225,7 @@ def __init__(self, arr,
                  **kwargs
                  ):
 
+        OffsetBox.__init__(self)
```

Comment:
looking back at the code: more precisely, the Artist class defines the `stale` property, and so if you want to set a value to that property during your constructor, then you need to construct your parent classes first. So these changes make sense. I just hope we didn't have code anywhere that depended on the older ordering